src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tasks that automate the job of moving projects in and out of symbolicator's low priority queue based
+on symbolication metrics stored in Redis.
+
+This has three major tasks, executed in the following general order:
+1. Scan for new suspect projects in Redis that need to be checked for LPQ eligibility. Triggers 2 and 3.
+2. Determine a project's eligibility for the LPQ based on their recorded metrics.
+3. Remove some specified project from the LPQ.
+"""
+
+import logging
+
+import sentry_sdk
+
+from sentry.processing.realtime_metrics import realtime_metrics_store
+from sentry.processing.realtime_metrics.base import BucketedCount
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.tasks.symbolicator.scan_for_suspect_projects",
+    queue="symbolicator.compute_low_priority_queue",
+    ignore_result=True,
+    soft_time_limit=10,
+)  # type: ignore
+def scan_for_suspect_projects() -> None:
+    _scan_for_suspect_projects()
+
+
+def _scan_for_suspect_projects() -> None:
+    suspect_projects = set()
+    for project_id in realtime_metrics_store.get_lpq_candidates():
+        suspect_projects.add(project_id)
+        compute_lpq_eligibility(project_id)
```

Comment:
Don't you want to `compute_lpq_eligibility.apply_async()` here?  IIUC this will run it inline.

Reply:
but that makes the deleted thing problematic, because you kind off need to wait until all the projects are done.  hum

Reply:
thanks for catching this, i messed up a detail here. 

> but that makes the deleted thing problematic, because you kind off need to wait until all the projects are done. 

i think this is okay: the purpose of the deleted block in this function is to prune the projects we definitely know shouldn't be in the queue. however, the bit i forgot is that this means `compute_lpq_eligibility` needs to remove the suspect project it's evaluating if it isn't eligible.